packages/website/src/pages/ContentPage/DocPage.tsx

```diff
@@ -158,6 +192,35 @@ function DocPage({ file, renderStackOffset = 0 }: IDocPageProps) {
 
   return (
     <div style={{ maxWidth: '50rem' }}>
```

Comment:
Since the rendering always take place in the `ContentPage` instead of globally, maybe you don't need the redux for storing such state. Just storing the state in the `ContentPage` should be enough :)

Reply:
okay, I was able to move it all to the FileAction view